src/api/app/models/package.rb

```diff
@@ -859,6 +859,17 @@ def rev
     dir_hash['rev']
   end
 
+  def self.current_revision(project, package)
+    dirhash = Directory.hashed(project: project, package: package)
```

Comment:
Hm, what do you mean exactly? This is exactly the code how it was before, just moved to the model ...